poster/src/poster.ts

```diff
@@ -46,11 +46,13 @@ export async function main(
 
     const trxData = buildTrxData(feedItems, functionSig);
     const gasEstimate = await web3.eth.estimateGas({data: trxData, to: viewAddress});
+    // Make gas estimate safer by 50% adjustment
+    const gastEstimateAdjusted = Math.floor(gasEstimate * 1.5);
```

Comment:
I guess there are two `GAS_ADJUSTMENT`s

Reply:
is that okay? I thought that keeping adjustment inside the `catch` clause if posting fails is useful